src/test/java/dk/sdu/se_f22/contentmodule/management/ManagementTest.java

```diff
@@ -0,0 +1,71 @@
+package dk.sdu.se_f22.contentmodule.management;
+
+import dk.sdu.se_f22.contentmodule.management.Data.*;
+import org.junit.jupiter.api.Test;
+import org.postgresql.util.PSQLException;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ManagementTest {
+
+    @Test
+    void create() {
+        try {
+            int id = Management.Create("This is a test HTML page");
+            assertEquals("This is a test HTML page", Management.getPageString(id));
```

Comment:
Failing test

Reply:
yeah but thats because it cant connect to the database :(